### PR TITLE
fix test log info

### PR DESCRIPTION
**Pull request by tanshanshan**
**What this PR does / why we need it**:

fix test log info

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:

**Release note**:

```release-note
NONE
```

### CODE REVIEW

**Review by MHBauer — APPROVED** [review]
/lgtm

File: staging/src/k8s.io/client-go/util/certificate/certificate_store_test.go [hunk 1]
```diff
@@ -69,16 +69,16 @@ func TestUpdateSymlinkNewFileNotExist(t *testing.T) {
 		pairNamePrefix: "kubelet",
 	}
 	if err := s.updateSymlink(oldPairFile); err != nil {
-		t.Errorf("Got %v, wanted successful update of the symlink to point to %q", err, oldPairFile)
+		t.Errorf("Got err %v, wanted successful update of the symlink to point to %q", err, oldPairFile)
```
Comment (nikhita) [inline staging/src/k8s.io/client-go/util/certificate/certificate_store_test.go]: Is the "err" needed in the message? If we are adding that, can we change it to "error"?

Reads better that way. :)

**Comment by nikhita** [comment]
/assign @liggitt 
for approval

File: staging/src/k8s.io/client-go/util/certificate/certificate_store_test.go (hunk 1 above)
Comment (tanshanshan) [reply staging/src/k8s.io/client-go/util/certificate/certificate_store_test.go]: DONE. thanks

**Pull request merged**